Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Build the source code
+FROM node:16-alpine AS builder
+RUN apk add --no-cache libc6-compat git python3 py3-pip make g++
+WORKDIR /app
+COPY . .
+
+# install deps
+RUN yarn install
+
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN yarn build
+RUN yarn export
+
+
+# Production image, copy out dir and run serve
+FROM node:16-alpine AS runner
+WORKDIR /app
+
+ENV NODE_ENV production
+# Uncomment the following line in case you want to disable telemetry during runtime.
+ENV NEXT_TELEMETRY_DISABLED 1
```

Comment:
Do we need it a second time?

Reply:
yep it seems that its needed, I used [this repo as a template](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile) because its recommended in[ the nextjs docs](https://nextjs.org/docs/deployment#docker-image)

Reply:
I mean it's already set on line 13 and we don't call any Next scripts after this.